post-scan-actions/aws-python-slack-notification/README.md

```diff
@@ -0,0 +1,213 @@
+# Post Scan Action - Slack Notification
+
+After a scan occurs, this example Lambda function sends out a notification to Slack using Amazon Lambda when a malicious file got detected.
+
+## Prerequisites
+
+1. **Install supporting tools**
+    - Install the AWS command line interface (CLI). See [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) for details.
+    - Install GNU Make if you don't want to use the AWS CLI. See [GNU Make](https://www.gnu.org/software/make/) for download information.
+2. **Configure Slack Webhook App**
+    - Create a Slack Channel to receive the notification
+    - Go to App Directory > Search `Incoming WebHooks`.
+    - Click on `Incoming WebHooks`, after click in "Add to Slack"
+    - Choose the Channel to receive the notification
+    - Copy Webhook URL
+    - Enter the Description of your WebHook.
+    -  Enter the Name of your WebHook If you want any customize name default it will be `incoming-webhook`.
+    - If you want any custom icon to add that in Customize Icon section.
```

Comment:
I think this way will be better right?

Enter the Name of the Slack WebHook, by default it will use 'incoming-webhook' if you prefer you can customize the name.

Reply:
That's better, but it needs a little more punctuation:
Enter the Name of the Slack WebHook, by default it will use 'incoming-webhook'; if you prefer, you can customize the name.

Reply:
sounds good to me